include/rmm/detail/cnmem.h

```diff
@@ -0,0 +1,264 @@
+/* ********************************************************************** 
+ * Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
```

Comment:
Why is this being added? cnmem just went away, right?

Reply:
That's strange.

Reply:
Removed